JapaneseFilter/sections/specific.txt

```diff
@@ -36,6 +36,7 @@ news24.jp,michill.jp,fnn.jp,bengo4.com,zigsow.jp,navitime.com,cazual.shufu.co.jp
 ||feed.mikle.com^$domain=blog.livedoor.jp|crx7601.com|fate-rpg.jp|gadgetlife2ch.blomaga.jp|gtoys.blog.fc2.com|h1g.jp|hdouga.com|ibarakinews.jp|obutu.com|totalwar.doorblog.jp|xn--bckadddb2a0d4dnc8dwhngdtb58aya0l4a2om843g5npczp7dwh5g.com|xn--cckea5a6cidcbh6ce7ghug17a2ge3aht3nwigef51658aw7kd.com
 !
 paypaymall.yahoo.co.jp##.YCA
+bazurecipe.com###sp_all_header
```

Comment:
Probably an existing rule `##.code-block` can be used and is safe here, but please double-check.

Reply:
`##.code-block`is working fine.